packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -762,7 +759,7 @@ export class CommonFrontendContribution implements FrontendApplicationContributi
         });
 
         commandRegistry.registerCommand(CommonCommands.OPEN_VIEW, {
-            execute: () => this.quickInputService?.open(this.quickView?.prefix)
+            execute: () => this.quickInputService?.open(QuickViewService.PREFIX)
```

Comment:
The following command does not work for me, likely related to changes in `quick-view-service`.
Triggering the command prompts you to choose a view from the list, but upon selection the view is never opened.

Reply:
I don't think it's the command: it also doesn't work when typing "view " in the `F1` quick access directly.

Reply:
Found and fixed.